mousestyles/behavior/metrics.py

```diff
@@ -0,0 +1,153 @@
+
+    df = df[strain_cond & mouse_cond & day_cond]
+
+    if df.shape[0] == 0:
+        raise ValueError("No consumption data available for specified mouse-day.")
+
+    if feature is 'W':
+        col_name = 'Water'
+    elif feature is 'F':
+        col_name = 'Food'
+    elif feature is 'L':
+        col_name = 'Distance'
+    else:
+        raise ValueError("Feature must be one of {'F', 'W', 'L'}.")
+
+    return df[col_name].sum()
+
+
+def create_intervals(activity_type, strain, mouse, day):
+    r"""Returns an interval object on a certain mouse-strain-day
+    for a given activity type.
+
+    Parameters
+    ----------
+    activity_type : str
+        String specifying activity type {"AS", "F", "IS", "M_AS", "M_IS", "W"}
+    strain : int
+        Integer representing the strain of the mouse
+    mouse : int
+        Integer representing the specific mouse
+    day : int
+        Integer representing the day to produce the metrics for
+
+    Returns
+    -------
+    An intervals object `intervals` for the given activity type and
+    mouse-strain-day.
+
+    Examples
+    --------
+
+    >>> ints=behavior.create_intervals(activity_type = 'F', strain = 0
+                                       , mouse = 0, day = 0)
+    >>> print(sum(ints))
+
+    """
+
+    # Load intervals by activity type
+    l_ints = data.load_intervals(str(activity_type))
+
+    # Subset to M*2 array of  (start, stop) intervals based on
+    # specific mouse, strain and day
+    l_ints = l_ints[(l_ints.strain == strain) &
+                    (l_ints.mouse == mouse) &
+                    (l_ints.day == day)][['start', 'stop']]
+
+    # Create and return the intervals object
+    return(intervals.Intervals(l_ints))
+
+
+def create_collapsed_intervals(intervals, bout_threshold):
+    r"""Returns a collapsed interval object (within a
+    given threshold) on a certain mouse-strain-day for a given activity type.
+
+    Parameters
+    ----------
+    intervals : interval object on a certain mouse-strain-day
+    for a given activity type
+    bout_threshold: : float
+        Float representing the time threshold to use for collapsing separate
+        events into bouts
+
+    Returns
+    -------
+    An intervals object `intervals` for the given activity type and
+    mouse-strain-day within a bout threshold
+
+    Examples
+    --------
+    >>> ints=behavior.create_intervals('F', strain = 0
+                                       , mouse = 0, day = 0
+                                       , bout_threshold = 0.001)
+    >>> collapsed_ints=behavior.create_collapsed_intervals(intervals = ints
+                                                    , bout_threshold = 0.001)
+    >>> print(collapsed_ints)
+
+    """
+
+    # Create and return the collapsed intervals object.
+    # Need to copy the original intervals object first else the collapse method
+    # will overwrite it as a collapsed interval object
+
+    return(intervals.copy().connect_gaps(eps=bout_threshold))
```

Comment:
Remove the bracket around `return` here i.e. change:
` return(intervals.Intervals(l_ints))` =  `return intervals.Intervals(l_ints)`